### PR TITLE
Added constexpr check to magic number checker.

### PR DESCRIPTION
**Pull request by jens-auer**
Added constexpr check to MagicNumberCheck to not report violations on constexpr definitions.

### CODE REVIEW

**Comment by jens-auer** [comment]
Could somebody provide some detail why the build fails? The log just says "Couldn't start SonarQube", which is not very helpful. I would fix the issue, but without any information this will be hard.

File: cxx-checks/src/test/resources/checks/magicNumber.cc [hunk 1]
```diff
@@ -43,13 +45,19 @@ int someData[] = {
 
 static const int i1 = 10;     // Compliant
 int i2 = 11;                  // Non-Compliant
+constexpr int compliant1 = 10;
```
Comment (guwirth) [inline cxx-checks/src/test/resources/checks/magicNumber.cc]: Maybe you can add //compliant -or- // non-compliant also to your new lines?

**Comment by guwirth** [comment]
Think this was an Travis CI problem. We had this also in the past.
Just make a change rebase, squash and try again.

**Comment by jens-auer** [comment]
Hi,

I've added the Compliant comments to the cpp test file.

**Pull request merged**